### PR TITLE
DOCSUP-71148: Fixes in function ref texts (part 4)

### PR DESCRIPTION
**Pull request by shamshinds**
https://st.yandex-team.ru/DOCSUP-71148

Function ref updates:

* text fixes and improvements;
* add examples;
* add USER_INFO function

### CODE REVIEW

File: lib/dl_formula_ref/dl_formula_ref/categories/window.py [hunk 1]
```diff
@@ -80,34 +83,27 @@
...
-                    "`SUM(SUM([Sales] BEFORE FILTER BY [Date])` expression are added.\n"
-                    "    - The `Date` dimension is added to the **Filters** section.\n"
-                    "\n"
-                    "    This will result in an error because the `Date` dimension isn't used to "
-                    "build the chart.\n"
+                    "   Examples:\n"
+                    "   * Correct formula: `RANK(SUM([Profit]) WITHIN [Category])` in the chart "
+                    "with grouping by the `[Order Date]` and `[Category]` dimensions. "
+                    "`[Category]` is used when building the chart.\n"
+                    "   * Valid formula: `RANK(SUM([Profit]) WITHIN [Category])` in the chart with "
```
Comment (MCPN) [inline lib/dl_formula_ref/dl_formula_ref/categories/window.py:96]: Probably should be `Correct formula` for consistency?

File: lib/dl_formula_ref/dl_formula_ref/locales/en/LC_MESSAGES/dl_formula_ref_dl_formula_ref.po
Comment (MCPN) [inline lib/dl_formula_ref/dl_formula_ref/locales/en/LC_MESSAGES/dl_formula_ref_dl_formula_ref.po:1]: Missing `.mo` changes

**Review by MCPN — CHANGES_REQUESTED** [review]

File: lib/dl_formula_ref/dl_formula_ref/locales/en/LC_MESSAGES/dl_formula_ref_dl_formula_ref.po
Comment (shamshinds) [reply lib/dl_formula_ref/dl_formula_ref/locales/en/LC_MESSAGES/dl_formula_ref_dl_formula_ref.po:1]: There are no changes after the launch of `i18n-binaries`

File: lib/dl_formula_ref/dl_formula_ref/categories/window.py (hunk 1 above)
Comment (shamshinds) [reply lib/dl_formula_ref/dl_formula_ref/categories/window.py:96]: fixes

**Review by MCPN — APPROVED** [review]

**Comment by MCPN** [comment]
`main / 🐍[pytest][ext_public]lib/dl_connector_metrica:ext (pull_request_target) Failing after 2m` - unrelated to PR content

**Pull request merged**